nautilus_trader/model/orders/stop_market.pyx

```diff
@@ -337,12 +337,12 @@ cdef class StopMarketOrder(Order):
             client_order_id=init.client_order_id,
             order_side=init.side,
             quantity=init.quantity,
-            trigger_price=Price.from_str_c(init.options["trigger_price"]),
+            trigger_price=Price.from_str_c(init.options['trigger_price']),
```

Comment:
Single quotes?

Reply:
Good catch, probably mistake. Fixed it here https://github.com/nautechsystems/nautilus_trader/pull/1650/commits/7c7102f8d8e562ebcf70474133688729a25ea8d3